### PR TITLE
Implement a "download" task using a new container

### PR DESCRIPTION
**Pull request by amatissart**
Changes in invoke configuration:
* `osm_file` is replaced with `osm.file` and `osm.url`
* `addresses.oa_file` is replaced with `addresses.oa.file` and `addresses.oa.download`
* `addresses.bano_file` is replaced with `addresses.bano.file` and `addresses.bano.url`

Note that the "download" task will fail if the pipeline does not use docker-compose and some downloads are enabled in the configuration.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,5 +42,6 @@ services:
 
 
 volumes:
+  dataosm:
```
Comment (antoine-de) [inline docker-compose.yml]: data_osm would be clearer

File: download/tasks.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ctx.run(f"mkdir -p {path.dirname(output_file)}")
+    gzip_file = f"{output_file}.gz"
+    ctx.run(f"wget --progress=dot:giga {bano_url} -O {gzip_file}")
+    ctx.run(f"gunzip -f {gzip_file}")
+
+@task
+def download_oa(ctx, oa_files):
+    temp_dir = tempfile.mkdtemp()
+
+    oa_files = oa_files.split(',')
```
Comment (antoine-de) [inline download/tasks.py:23]: why don't you use a list of files as cli args ?

File: invoke.yaml [hunk 3]
```diff
@@ -38,12 +39,16 @@ dataset: default
 #   ## If present will use osm2mimir to import the poi
 #   osm:
 
-# addresses:
-#   # PATH to the openaddresses file
-#   oa_file:
+addresses:
+  oa:
+    file: # Ignored 'download' is defined
```
Comment (antoine-de) [inline invoke.yaml]: **if** defined no ?

File: docker_settings.yaml [hunk 4]
```diff
@@ -1,10 +1,121 @@
-# example file to run mimir in docker, the file path must be changed
-osm_file: /data/osm/luxembourg-latest.osm.pbf
+# example file to run mimir in docker
+
```
Comment (antoine-de) [inline docker_settings.yaml:2]: side note but can you add 
```
geocoder_tester_region: luxembourg
```

to the file ? this way it's a complete luxembourg example

File: download/tasks.py (hunk 2 above)
Comment (amatissart) [reply download/tasks.py:23]: No good reason. It seems just a bit cumbersome to build the parameters `--oa-files <file1> --oa-files <file2> ...` in the calling command when a list is already set in a .yml file.

**Review by antoine-de — APPROVED** [review]

**Pull request merged**